tools/stats/gsummary.xml.groups

```diff
@@ -52,11 +53,12 @@ This tool computes basic summary statistics on a given column, or on an expressi
 
 .. class:: infomark
 
-**TIP:** Most functions (like *abs*) take only a single expression. *log* can take one or two parameters, like *log(expression,base)* 
+**TIP:** Most functions (like *abs*) take only a single expression. *log* can take one or two parameters, like *log(expression,base)*
 
 Currently, these R functions are supported: *abs, sign, sqrt, floor, ceiling, trunc, round, signif, exp, log, cos, sin, tan, acos, asin, atan, cosh, sinh, tanh, acosh, asinh, atanh, lgamma, gamma, gammaCody, digamma, trigamma, cumsum, cumprod, cummax, cummin*
 
 .. |INFO| image:: ./static/images/icon_info_sml.gif
 
-</help>
+  </help>
+  <citations />
```

Comment:
I wouldn't add the empty citations block. @blankenberg do you think this tool is correlated with any particular Galaxy paper?